src/schemas/686c-674/schema.js

```diff
@@ -0,0 +1,941 @@
+import {
+  countries, 
+  states50AndDC, 
+  suffixes,
+} from '../../common/constants.js';
```

Comment:
not sure if `.js` is needed on the end of this import.

Reply:
weird, VS code added this when I formatted the doc. Removed.